admin/authentication/local-strategy.js

```diff
@@ -64,23 +74,19 @@ async function saveInvalidLogonEvent (logonEvent, message) {
   try {
     logonEvent.errorMsg = message
     logonEvent.isAuthenticated = false
-    await logonEvent.save()
+    await adminLogonEventDataService.sqlCreate(logonEvent)
   } catch (error) {
-    console.log('Failed to save logon event: ' + error.message)
+    winston.log('Failed to save logon event: ' + error.message)
```

Comment:
IIRC you are missing the log level in this call.  Either use `winston.log('info|warn|error|debug', 'message')`
or one of the convenience methods such as `winston.info`